terraform/main.tf

```diff
@@ -0,0 +1,42 @@
+resource "juju_application" "grafana-agent-k8s" {
+  name  = "grafana-agent-k8s"
+  model = var.model_name
+
+  charm {
+    name    = "grafana-agent-k8s"
+    channel = var.channel
+    base    = "ubuntu@22.04"
```

Comment:
In Grafana's `charmcraft.yaml` we have:
```
bases:
  - build-on:
      - name: "ubuntu"
        channel: "20.04"
    run-on:
      - name: "ubuntu"
        channel: "20.04"
```
Do we need to specify the base at all here?

Reply:
When I check the charmcraft.yaml, it uses ubuntu 22.04:
```
type: charm
bases:
  - build-on:
    - name: "ubuntu"
      channel: "22.04"
      architectures: ["amd64"]
    run-on:
    - name: "ubuntu"
      channel: "22.04"
      architectures: ["amd64"]
```

Reply:
I think w can get rid of base.

Reply:
Done